### PR TITLE
Added Kyverno policies and README.md

### PR DESCRIPTION
**Pull request by mzeevi**
Added different policies to the `policies/` directory and made some changes to the existing ones. All the policies are also documented in the `README.md` file.

Signed-off-by: mzeevi <meytar80@gmail.com>

### CODE REVIEW

**Comment by mzeevi** [comment]
Test on a OpenShift cluster to see if everything works. I only tested what I could on `KinD`

cc @sahar2339 @nadavdolin

**Comment by mzeevi** [comment]
Test on a OpenShift cluster to see if everything works. I only tested what I could on KinD

/cc @sahar2339 @nadavdolin

File: policies/deny-configmaps.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        operator: Equals
+        value: "user-ca-bundle"
+    validate:
+      message: "Prevented modification of ConfigMaps that are called [user-ca-bundle] or that live in namespace [openshift-config]. This is in an effort to prevent harmful actions that may cause unintended consequences or affect the stability of the cluster."
+      deny:
+        conditions:
+          all:
+          - key: "{{ privileged.data.groups | parse_yaml(@) }}"
+            operator: AllNotIn
+            value: "{{ request.userInfo.groups }}"
```
Comment (nadavdolin) [inline policies/deny-configmaps.yml:35]: add condition also for privileged username

File: policies/deny-techpreviewupgrade.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - name: Disallow usgae of Tech Preview features
+    match:
+      any:
+      - resources:
+          kinds:
+          - config.openshift.io/*/FeatureGate
+    validate:
+      message: Managed Runb Kubernetes Service (RKS) Customers may not use TechPreviewNoUpgrade FeatureGate that could prevent any future ability to do an upgrade of their clusters.
+      deny:
+        conditions:
```
Comment (nadavdolin) [inline policies/deny-techpreviewupgrade.yml]: Add condition to allow privilege users

**Comment by nadavdolin** [comment]
Looks good to me /approve

**Review by nadavdolin — APPROVED** [review]

**Pull request merged**